src/App.js

```diff
@@ -1,16 +1,90 @@
-import React from 'react';
-
+import React from "react";
+import ReactDom from "react-dom";
```

Comment:
no need to import react dom here, its imported in the root of the project index.js